lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -129,17 +129,26 @@ def get_results(self, name_list: list[str], query: JSON, server_config) -> JSON:
         Returns:
             Paginated JSON object containing list of dataset values
         """
+
+        def convert(k: str, v: Any) -> Any:
+            if isinstance(v, str) and k in ("filter", "sort", "metadata"):
+                return [v]
+            elif isinstance(v, int):
+                return str(v)
+            else:
+                return v
+
         results: list[JSON] = []
-        offset = query.get("offset", 0)
+        offset = int(query.get("offset", 0))
```

Comment:
Since `int(0) == int("0")` it really doesn't matter, and this was more compact and arguably a little quicker. I did consider that, and decided it was excessively unnecessary consistency. Do I want to reconsider that? I don't know...